docs/en/siem/machine-learning.asciidoc

```diff
@@ -20,9 +20,9 @@ image::ml-ui.png[]
 [float]
 [[manage-jobs]]
 === Manage machine learning jobs
-For users with the `ml_admin` role, the `Anomaly Detection` interface within
-the main navigation header can be used for for viewing, starting, and stopping
-SIEM machine learning jobs.
+For users with the `machine_learning_admin` role, the `Anomaly Detection`
```

Comment:
I'm sure you're aware, but with #62396 this is changing to `ML job settings` and will only be visible on the Detections page.